packages/cli/commands/project/installDeps.js

```diff
@@ -0,0 +1,73 @@
+const {
+  installPackages,
+  getProjectPackageJsonLocations,
+} = require('../../lib/dependencyManagement');
+const { logger } = require('@hubspot/local-dev-lib/logger');
+const { EXIT_CODES } = require('../../lib/enums/exitCodes');
+const { getProjectConfig } = require('../../lib/projects');
+const { promptUser } = require('../../lib/prompts/promptUtils');
+const path = require('path');
+const { i18n } = require('../../lib/lang');
+
+const i18nKey = `commands.project.subcommands.installDeps`;
+
+exports.command = 'install-deps [packages..]';
+// Intentionally making this null to hide command
+exports.describe = null;
+// exports.describe = uiBetaTag(i18n(`${i18nKey}.help.describe`), false);
+exports.builder = yargs => {
```

Comment:
Big nit, but in all of our other commands we've been placing the builder at the bottom of the file. I'd vote to keep consistency in that to make things easier to quickly find

Reply:
https://github.com/HubSpot/hubspot-cli/pull/1133/commits/2a9e7834c1260fd0cd49ac2908f6547f61804361